packages/lb-components/src/types/data.d.ts

```diff
@@ -1,10 +1,23 @@
 import { ESubmitType } from '@/constant';
+import { TMatrix13Tuple, TMatrix14Tuple } from '@labelbee/lb-utils';
 
 /** 标注文件对象 */
+export interface ICalib {
```

Comment:
comment for ICalib to explain P R T

Reply:
I will add a document to describe the pointCloud Terminology